### DISCUSSION TITLE
Why is this happening??

### DISCUSSION THREAD

**Discussion by Bereket-Desbele** [espnet/espnet#4874] [Q&A] [answered]
I've undergone some experiments with the espnet-1 ASR (hybrid ctc-attention) and every time i use specaug the validation accuracy & validation CER gets higher than the Main/accuracy & Main/CER. While the validation/loss gets lower than Main/loss. Isn't SpecAug supposed to avoid overfitt
![acc with SpecAug](https://user-images.githubusercontent.com/41731956/213149358-5eb42596-7b3e-4054-ac14-c2ff0c0e93ae.png)
![acc without SpecAug](https://user-images.githubusercontent.com/41731956/213149384-7b533c23-8834-457b-8938-22be00c372b2.png)
![cer with SpecAug](https://user-images.githubusercontent.com/41731956/213149392-7a3c067c-8173-4475-bb3f-f1154e870a49.png)
![cer without SpecAug](https://user-images.githubusercontent.com/41731956/213149395-088e506d-3ac1-4ee0-a42f-08f4f300365b.png)
![loss with SpecAug](https://user-images.githubusercontent.com/41731956/213149401-adc1e3a5-2495-4664-b465-4fced1301ca9.png)
![loss without SpecAug](https://user-images.githubusercontent.com/41731956/213149411-61d98621-480c-4124-8d77-cc2cbbef27e9.png)
ing?? What does this mean? and Why is it happening? here are some picture results..

**Comment by Bereket-Desbele** [comment]
Is it because the data is small (Its 30hrs). Or is it because the training and validation data both have similar trend that its doing better in validation??

**Comment by sw005320** [comment, marked as answer]
This is because the specaug is only applied during the training phase, making the prediction more difficult than that in the inference phase.

**Reply by Bereket-Desbele** [reply]
So, is this a valid result? If not what are we supposed to do to fix it?? And Many thanks in advance.

**Reply by sw005320** [reply]
This is a valid result.

**Reply by Bereket-Desbele** [reply]
Kudos for the educational tip. & Thanks again, Dr.
